src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py

```diff
@@ -0,0 +1,85 @@
+from copy import deepcopy
+from dataclasses import dataclass
+from functools import lru_cache
+from types import MappingProxyType
+from warnings import warn
+
+import os
+import torch
+import torch.overrides
+
+from torch.fx import GraphModule
+
+from openvino.frontend import FrontEndManager
+from openvino.frontend.pytorch.fx_decoder import TorchFXPythonDecoder
+from openvino.runtime import Core, Type, PartialShape, serialize
+
+from typing import Callable, Optional
+
+
+import numpy as np
+def openvino_compile(gm: GraphModule, *args, model_hash_str : str = None):
+    core = Core()
+
+    device = 'CPU'
+
+    if os.getenv("OPENVINO_DEVICE") is not None:
+        device = os.getenv("OPENVINO_DEVICE")
+        assert device in core.available_devices, "Specified device " + device + " is not in the list of OpenVINO Available Devices"
+
+    file_name = None
+    cache_root = "./cache/"
+    if os.getenv("OPENVINO_TORCH_CACHE_DIR") is not None:
+        cache_root = os.getenv("OPENVINO_TORCH_CACHE_DIR")
```

Comment:
Understood that env vars is the most easiest way to control everything, but still in this case we can pass it as backend argument.

Reply:
As discussed, will implement this after pre-release